From 29bfe617802827d19f4bf3e4e26bcef9554dddca Mon Sep 17 00:00:00 2001
From: Danny Kukawka <danny.kukawka@web.de>
Date: Fri, 9 Apr 2010 14:41:12 +0200
Subject: [PATCH 25/33] added some more HP models to get brightness keys working

Added some more HP models to get brightness keys working:
- EliteBook 8440w
- EliteBook 8740w
- EliteBook 8540p

They need laptop_panel.brightness_in_hardware=false
---
 .../10freedesktop/10-laptop-panel-hardware.fdi     |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/fdi/information/10freedesktop/10-laptop-panel-hardware.fdi b/fdi/information/10freedesktop/10-laptop-panel-hardware.fdi
index 07fe031..1b79e1e 100644
--- a/fdi/information/10freedesktop/10-laptop-panel-hardware.fdi
+++ b/fdi/information/10freedesktop/10-laptop-panel-hardware.fdi
@@ -37,7 +37,7 @@
       </match>
 
       <match key="/org/freedesktop/Hal/devices/computer:system.hardware.vendor" string="Hewlett-Packard">
-        <match key="/org/freedesktop/Hal/devices/computer:system.hardware.product" contains_outof="4310s;4410s;4411s;4415s;4416s;4510s;4515s;4710s;Compaq 5X5;Compaq 515;Compaq 516;Compaq 615;HP Mini 5102">
+        <match key="/org/freedesktop/Hal/devices/computer:system.hardware.product" contains_outof="4310s;4410s;4411s;4415s;4416s;4510s;4515s;4710s;Compaq 5X5;Compaq 515;Compaq 516;Compaq 615;HP Mini 5102;EliteBook 8440w;EliteBook 8740w;EliteBook 8540p">
           <merge key="laptop_panel.brightness_in_hardware" type="bool">false</merge>
         </match>
       </match>
-- 
1.7.1

